app/src/main/java/com/gh4a/activities/issues_label/IssuesLabelService.java

```diff
@@ -0,0 +1,63 @@
+package com.gh4a.activities.issues_label;
+
+import android.app.Activity;
+import android.view.View;
+import com.gh4a.Gh4Application;
+import com.gh4a.R;
+import com.gh4a.utils.rx.RxTools;
+import org.eclipse.egit.github.core.Label;
+import org.eclipse.egit.github.core.RepositoryId;
+import org.eclipse.egit.github.core.service.LabelService;
+import java.net.URLEncoder;
+import io.reactivex.Observable;
+
+public class IssuesLabelService {
+    public static final int DELETE_ISSUE_LABEL = 0;
+    public static final int EDIT_ISSUE_LABEL = 1;
+    public static final int ADD_ISSUE_LABEL = 2;
+
+    public static Observable deleteIssueLabel(Activity activity, View rootView, String repoOwner, String repoName, String labelName) {
+        String errorMessage = activity.getString(R.string.issue_error_delete_label, labelName);
+
+        return Observable.fromCallable(() -> {
```

Comment:
Why Observable here? I feel like it would be better to simply use Completable since we don't return anything.